internal/template.go

```diff
@@ -17,6 +17,8 @@ const (
 	CRITICAL = "critical"
 )
 
+var ValidSeverityLevels = []string{LOW, MEDIUM, HIGH, CRITICAL}
```

Comment:
suggestion: Having the severities defined here and used for both the `template` and `build` command feels a bit unnatural to me.

What do you think about having these defined in a separate file, called `types`, `constants`, or `severities`? That way, we don't associate severities with a particular command.